api/authorization-api-1_0.md

```diff
@@ -1059,17 +1025,21 @@ The following payload defines a request for the resources of type `account` on w
   },
   "resource": {
     "type": "account"
+  },
+  "page": {
+    "type": "token",
+    "size": 2
   }
 }
 ~~~
-{: #resource-search-request-example title="Example Request"}
-
-## The Resource Search API Response {#resource-search-response}
-
-The response is a paged array of Resources.
+{: #search-pagination-token-initial-request title="Example initial Search Request with token-based pagination"}
 
 ~~~ json
 {
+  "page": {
+    "next_token": "a3M9NDU2O3N6PTI=",
+    "count": 2
```

Comment:
Do we need `count` if the length of the `results` array provides the real number?